demo.js

```diff
@@ -0,0 +1,10 @@
+const something = "im a bad thing"
+const breakingChange = something.notValid
+
+const newThing = "a thing ive never seen before"
```

Comment:
Question: I haven’t seen that syntax used like this. I’m curious, why did you choose to use it here?